doc/indexing.rst

```diff
@@ -108,6 +111,13 @@ use them explicitly to slice data. There are two ways to do this:
         # index by dimension coordinate labels
         arr.sel(time=slice('2000-01-01', '2000-01-02'))
 
+3. Use the :py:meth:`~xray.DataArray.isel_points` method:
+
+    .. ipython:: python
+
+        # index by integer array indices
+        arr.isel_points(space=[0, 1], dim='points')
```

Comment:
A more compelling example would show actual pointwise indexing, instead of only indexing along one dimension.